nautobot/dcim/views.py

```diff
@@ -247,8 +247,8 @@ def get_extra_context(self, request, instance):
             .select_related("parent", "location_type")
```

Comment:
Would it make sense to move the `.restrict()` call into the ObjectsTablePanel class logic so that we don't need to repeat it throughout?